tests/integration/test_ttl_multilevel_group_by/test.py

```diff
@@ -0,0 +1,76 @@
+import time
+import pytest
+from helpers.cluster import ClickHouseCluster
+
+cluster = ClickHouseCluster(__file__)
+node = cluster.add_instance("node", with_minio=False)
+
+
+@pytest.fixture(scope="module")
+def started_cluster():
+    try:
+        cluster.start()
+        yield cluster
+    finally:
+        cluster.shutdown()
+
+def test_two_stage_ttl_group_by(started_cluster):
+    node = cluster.instances["node"]
+
+    node.query("DROP TABLE IF EXISTS test_ttl_group_by")
+
+    # Create table with two TTL GROUP BY rules:
+    #   1. After 13 seconds, aggregate by user_id, item_id using SUM
+    #   2. After 18 seconds, aggregate by user_id using SUM
+    node.query("""
+    CREATE TABLE test_ttl_group_by (
+        event_date DateTime,
+        user_id Int32,
+        item_id Int32,
+        val Float32,
+        rolled_by_item_id DateTime DEFAULT event_date,
+        rolled_by_user_id DateTime DEFAULT event_date
+    ) ENGINE = MergeTree()
+        PRIMARY KEY (event_date, user_id, item_id)
+        TTL rolled_by_item_id + INTERVAL 13 SECOND
+            GROUP BY event_date, user_id, item_id
+            SET val = sum(val),
+                rolled_by_item_id = max(rolled_by_item_id) + INTERVAL 10 YEAR,
+        rolled_by_user_id + INTERVAL 18 SECOND
+            GROUP BY event_date, user_id
+            SET val = sum(val),
+                rolled_by_user_id = max(rolled_by_user_id) + INTERVAL 10 YEAR
+        SETTINGS merge_with_ttl_timeout = 0;
+    """)
+
+    # Insert 10 rows with current timestamp, evenly distributed across 2 user_ids and 3 items
+    node.query("""
+        INSERT INTO test_ttl_group_by
+        SELECT
+            now() AS event_date,  -- Указываем фиксированное время
```

Comment:
```suggestion
            now() AS event_date,  -- Указываем фиксированное время
```

Could you please either translate it to English or remove it?